packages/cli/src/databases/entities/User.ts

```diff
@@ -155,4 +155,14 @@ export class User extends WithTimestamps implements IUser {
 		const { password, apiKey, mfaSecret, mfaRecoveryCodes, ...rest } = this;
 		return rest;
 	}
+
+	createPersonalProjectName() {
+		if (this.firstName && this.lastName && this.email) {
+			return `${this.firstName} ${this.lastName} <${this.email}>`;
+		} else if (this.email) {
+			return `<${this.email}>`;
```

Comment:
@krynble This is what it generates for shell accounts.

Reply:
Good idea.

They're here: 

https://github.com/n8n-io/n8n/pull/8995/files#diff-0157cebf7fcd16044b39d250c31aa3f3ae85e14d1afe27f4afb7dc17b0999225R21-R37